File: WebstormProjects/my_first/dietelQuestions/chapter7/removeDuplicate.js


function noDuplicate(arr){
    let newArr = []
    let bool = false
    newArr.push(arr[0])
    for (let i = 0; i < arr.length; i++) {
        bool = false
        for (let j = 0; j <newArr.length; j++) {
            if(arr[i] === newArr[j]){
                bool = false;
              break
            }
            else if(arr[i] !== newArr[j]){
                bool = true
            }
        }
        if (bool) newArr.push(arr[i])
    }
    return newArr
}

let question = [2,3,3,4,1,1,12,4]
console.log(noDuplicate(question))

File: PycharmProjects/Practice/src/alx_learning/removeDuplicate.py


def noDuplicate(arr):
    newArr = []
    isTrue = False
    newArr.append(arr[0])
    for i in range(len(arr)):
        isTrue = False
        for j in range (len(newArr)):
            if arr[i] == newArr[j]:
                isTrue = False
                break
            elif arr[i] != newArr[j]:
                isTrue = True

        if (isTrue): newArr.append(arr[i])

    return newArr

def removeDuplicate(arr):
    tup = (arr[0])
    for i in range (len(arr)):
        tup = tup + (arr[i],)
    return tup



# question = [2,3,3,4,1,1,12,4]
# print(removeDuplicate(question))
tup = (1,2,4,5,1)
print(tup)